plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/BaseTransactionsTable.java

```diff
@@ -170,7 +171,7 @@ private static List<Transaction> loadNewTailBackward(
         boolean endOfHead = false;
 
         while (!endOfHead) {
-            Optional<List<DeltaLakeTransactionLogEntry>> results = getEntriesFromJson(entryNumber, transactionLogDir, fileSystem);
+            Optional<TransactionLogEntries> results = getEntriesFromJson(entryNumber, transactionLogDir, fileSystem, DEFAULT_TRANSACTION_LOG_MAX_CACHED_SIZE);
```

Comment:
you could maybe retrieve through `TransactionLogAccess` the `transactionLogMaxCachedSize` instead of using hardcoded values. 
I'm refferring to `DEFAULT_TRANSACTION_LOG_MAX_CACHED_SIZE`

Same for `DeltaLakeMetadata`

Reply:
I don't think its useful to cache transaction log in those contexts, so i've just set it to 0 bytes to be clearer